src/renderer/helpers/walletHelper.ts

```diff
@@ -130,6 +130,9 @@ export const getBnbAmountFromBalances = (balances: WalletBalances): O.Option<Ass
 export const getEthAmountFromBalances = (balances: WalletBalances): O.Option<AssetAmount> =>
   getAssetAmountFromBalances(balances, isEthAsset)
 
+export const getEVMAmountFromBalances = (balances: WalletBalances): O.Option<AssetAmount> =>
```

Comment:
This should replace getBscAmountFromBalances, getAvaxAmountFromBalances and getEthAmountFromBalances. so can they be deleted?

Reply:
maybe, id like to leave them there for now